cheroot/server.py

```diff
@@ -1693,16 +1697,17 @@ def tick(self):
                 return
             raise
 
-    def _get_interrupt(self):
+    @property
+    def interrupt(self):
+        """Return interrupt Exception instance."""
```

Comment:
This docstring is the one that will be present on the property, whereas previously the docstring would have provided indication about setting the exception. The docstring should be copied from the setter and the setter should probably have no meaningful docstring, maybe something like "setter for interrupt".

Reply:
Fair enough

Reply:
But still the docstring should describe the property/getter

Reply:
I'll try to rephrase that